types-fixup.mjs

```diff
@@ -437,6 +437,13 @@ const standardMaterialProps = [
     ['specularTint', 'boolean'],
     ['specularVertexColor', 'boolean'],
     ['specularVertexColorChannel', 'string'],
+    ['useSpecularityFactor', 'boolean'],
```

Comment:
Is this property needed if 'specularityFactor' is set to 0?

Reply:
Yes, I used this before to select the shader, but I've changed this to only be reflected in the material options, not in the material API.